website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -261,19 +263,20 @@ spec:
 
 </CodeBlockConfig>
 
--> **NOTE**: If TLS is enabled for external services registered through the Consul catalog, you must include the `caFile` parameter that points to the system trust store of the terminating gateway container.
+-> **NOTE**: If TLS is enabled for external services registered through the Consul catalog and you are not using [transparent proxy `Destinations`](#register-an-external-service-as-a-destination), you must include the `caFile` parameter that points to the system trust store of the terminating gateway container.
 By default, the trust store is located in the `/etc/ssl/certs/ca-certificates.crt` directory.
 Configure the `caFile` parameter to point to the `/etc/ssl/cert.pem` directory if TLS is enabled and you are using one of the following components:
  * Consul Helm chart 0.43 or older
  * Or an Envoy image with an alpine base image
+For Transparent Proxy Destinations, see the [guidance in the table above](#register-an-external-service-as-a-destination).
```

Comment:
<img width="965" alt="CleanShot 2022-08-22 at 12 03 57" src="https://user-images.githubusercontent.com/1034429/185998933-9d52e7cd-44cf-4e31-bf9b-65c636581f74.png">

The rendering is messed up here. Part of the note has text it shouldn't and the last bullet has text it shouldn't.